RecoTracker/FinalTrackSelectors/plugins/MultiTrackSelector.cc

```diff
@@ -561,10 +575,13 @@ void MultiTrackSelector::processMVA(edm::Event& evt, const edm::EventSetup& es,
     return;
   }
 
+  if(!useMVA_[selIndex] && !writeIt)return;
```

Comment:
Hi Matt,

the odd part was why it is crashing here.
Could you please confirm that you can run run2 MC with this code.